src/js/components/account/Account.jsx

```diff
@@ -21,6 +24,7 @@ export default class Account extends React.Component {
     render() {
         return (
             <div className="usa-da-account-page">
+                <MetaTags {...MetaTagHelper.federalAccountPageMetaTags} />
```

Comment:
If we're going to use react-helmet, why not make the titles dynamic (account/award title)?